tox.ini

```diff
@@ -50,3 +48,11 @@ deps =
   sphinx-rtd-theme==0.5.2
 commands =
   sphinx-build -b html docs/source docs/html
+
+[testenv:mypy]
```

Comment:
Tested locally, works as expected :)

Please also update `.circleci/config.yml` so that mypy check is also executed under CI!

Reply:
See https://app.circleci.com/pipelines/github/VirtusLab/git-machete (log in with Github)

Reply:
Added to CI! 

A little more rework was needed in `tox.ini` to handle running `mypy` on all supported versions (for local testing convenience) or a specific version (for CI tests).